lib/generators/react_on_rails/base_generator.rb

```diff
@@ -91,7 +91,8 @@ def add_yarn_dependencies
             babel-plugin-macros"
 
         puts "Adding CSS handlers"
-        run "yarn add css-loader css-minimizer-webpack-plugin mini-css-extract-plugin style-loader"
+        # css-minimizer-webpack-plugin v6.0.0 requires Node 18+
+        run "yarn add css-loader 'css-minimizer-webpack-plugin@<6.0.0' mini-css-extract-plugin style-loader"
```

Comment:
This is added in a [separate PR](https://github.com/shakacode/react_on_rails/pull/1598) with an updated changelog, with this in mind that it will be merged before this PR. I needed this to be fixed here as well so I added the same code here too.